arm/Microsoft.Compute/virtualMachines/.parameters/linux.autmg.parameters.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Must be referenced in Azure DevOps pipelines

Reply:
Done